huaweicloud/services/sms/resource_huaweicloud_sms_task.go

```diff
@@ -488,13 +566,157 @@ func resourceMigrateTaskRead(_ context.Context, d *schema.ResourceData, meta int
 		d.Set("enterprise_project_id", migTask.EnterpriseProjectId),
 		d.Set("migrate_speed", migTask.MigrateSpeed),
 	)
+
+	speedLimit, err := GetTaskSpeedLimit(smsClient, d.Id())
+	if err != nil {
+		return common.CheckDeletedDiag(d, err, "error retrieving task speed limit")
+	}
+	mErr = multierror.Append(mErr,
+		d.Set("speed_limit", flattenSmsTaskSpeedLimit(
+			utils.PathSearch("speed_limit", speedLimit, make([]interface{}, 0)).([]interface{}))),
+	)
+
+	configurations, err := GetTaskConfigurations(smsClient, d, d.Id())
+	if err != nil {
+		return common.CheckDeletedDiag(d, err, "error retrieving task configurations")
+	}
+	mErr = multierror.Append(mErr,
+		d.Set("migrate_type", utils.PathSearch("migrate_type", configurations, nil)),
+		d.Set("configurations", flattenSmsTaskConfigurations(
+			utils.PathSearch("configurations", configurations, make([]interface{}, 0)).([]interface{}))),
+	)
+
+	if migTask.State == "RUNNING" || migTask.State == "SYNCING" {
+		passphrase, err := GetTaskPassphrase(smsClient, d.Id())
+		if err != nil {
+			return common.CheckDeletedDiag(d, err, "error retrieving task passphrase")
+		}
+		mErr = multierror.Append(mErr,
+			d.Set("passphrase", utils.PathSearch("passphrase", passphrase, nil)),
+		)
+	}
```

Comment:
print log if `err` is not nil
```suggestion
		passphrase, err := GetTaskPassphrase(smsClient, d.Id())
		if err != nil {
			return common.CheckDeletedDiag(d, err, "error retrieving task passphrase")
		}
		mErr = multierror.Append(mErr,
			d.Set("passphrase", utils.PathSearch("passphrase", passphrase, nil)),
		)
```